### ISSUE TITLE
"Run Current Chunk" not working for bash on Windows 10

### ISSUE THREAD

**Issue by Michael-Beyeler** [rstudio/rstudio#2724]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.1.442
    OS Version      : Windows 10 64bit, April 2018 Update
    R Version       :  3.4.3 (Microsoft R Open, but that shouldn't matter)

### Steps to reproduce the problem
1. Create a new .Rmd document and save it somewhere
2. In it, write
`*tripplegraveaccent*{bash}`
`echo hello world`
`pwd`
`*tripplegraveaccent*`
3. Knit the document (this should work).
4. Click on "Run Current Chunk", which results in the error "/bin/bash: C:\Users\myuser\AppData\Local\Temp\RtmpwF2huT\chunk-code-6618365a64b8.txt: No such file or directory"

### Describe the behavior you expected

Basically, what I expected, is for "Run Current Chunk" to execute the bash code just as knitr does, which isn't the case. I guess Run Current Chunk points to a path where bash isn't present or something along the line of that. But I'm not sure.

Anyhow, the most likely case is that Run Current Chunk is broken at the moment, because knitr works just fine.

I'm grateful for any solution or workaround!

Best regards

**Comment by dfalty** [comment]
I'm able to reproduce this as well on the latest daily 1.2.616. I have WSL installed with Ubuntu and the knit works but I get the same error on running the chunk. I tried setting the engine.path for that chunk but that doesn't work either. Thanks for the report! It would be nice if we could support this setup on this ticket as well as the other one you put in. https://github.com/rstudio/rstudio/issues/2727

**Comment by ronblum** [comment]
Verified on multiple environments, using both `bash` and `sh`.
